package.json

```diff
@@ -6,7 +6,6 @@
   "author": "Discourse",
   "license": "GPL-2.0-only",
   "dependencies": {
-    "@fortawesome/fontawesome-free": "5.15.4",
```

Comment:
Is this intentional?

Reply:
Yup! See: https://github.com/discourse/discourse/pull/17218/commits/8d19adb781cac0cbc897ce07bb0659a99d312689